app/components/page-components/Chatbot/Commands/ChatbotCustomCommands.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+<div>
+  <!-- batch actions -->
+  <div class="flex flex--space-between padding--10">
+    <button
+      @click="openCommandWindow"
+      class="button button--action margin--10"
+    >
+      Add Command
+    </button>
+    <input
+      type="text"
+      class="chatbot__input--search width--auto margin--10"
+      placeholder="Search"
+    />
+  </div>
+
+  <!-- custom commands -->
+  <div class="padding--10">
+    <div v-if="commands.length === 0">
+      <h2>No custom commands. Click to add new.</h2>
+    </div>
+    <table v-else>
+      <thead>
+        <tr>
+          <th>Command</th>
+          <th>Response</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="(command, index) in commands"
+          :key="index"
+        >
+          <td> {{ $t(command.command) }} </td>
+          <td> {{ $t(command.response) }} </td>
```

Comment:
our i18n dictionaries are static. So use the template function only with constant strings like `$t('Command')`. DM if you need more information.